### DISCUSSION TITLE
Car wakes up every 9 minutes to charge for a few seconds

### DISCUSSION THREAD

**Discussion by mikey70** [teslamate-org/teslamate#1208] [Q&A]
New Teslamate user here, just installed last night, so I have no idea if this is an issue with the car, or if Teslamate could be causing this.  My "states" graph shows a consistent pattern: Car sleeps for 9 minutes, wakes up to charge for "a few seconds", then stays online for 16 minutes -- repeat.

Car is parked in the garage and plugged in.  Streaming API is enabled.  2021 Model S.

Any ideas?

![image](https://user-images.githubusercontent.com/47435855/104747907-50b30400-570e-11eb-93c5-50637b3bc345.png)

**Comment by corbosman** [comment]
Hey did you ever figure this out? I have the exact same problem. Charges for a few seconds, then falls asleep again. What charger do you have?

![Screenshot 2021-08-14 at 13 02 27](https://user-images.githubusercontent.com/1389007/129444332-a4e04e25-b009-4594-b100-383048b7e305.png)

**Reply by kimmilde** [reply]
This is a tesla SW bug

**Reply by corbosman** [reply]
@kimmilde do you have a reference i can read? It seems likely it's a SW bug as I've pretty much excluded everything else.

**Comment by corbosman** [comment]
The latest tesla software version seems to have fixed this problem for me. My car sleeps again while plugged in.
